src/model/WorkflowState.model.js

```diff
@@ -35,6 +36,8 @@ class WorkflowState extends Enumify {
   static STOPPING = new WorkflowState('stopping', mdiSkipNextCircle)
   static STOPPED = new WorkflowState('stopped', mdiStopCircle)
   static ERROR = new WorkflowState('error', mdiHelpCircle)
+  static INSTALLED = new WorkflowState('installed', mdiFolderPlus)
+  static UNINSTALLED = new WorkflowState('uninstalled', null)
```

Comment:
Note to self: remove this state, not in https://github.com/cylc/cylc-flow/blob/5e6ebe3cf1280e29fdc6b744d49271bb8f1cfec6/cylc/flow/workflow_status.py